### PR TITLE
DOCS-2549: Add deploy packages to 1000s of machines

### PR DESCRIPTION
**Pull request by npentrel**
https://docs-test.viam.dev/3361/how-tos/deploy-packages/index.html

### CODE REVIEW

File: docs/how-tos/deploy-packages.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+      "module_id": "viam-labs:speech",
+      "version": "0.5.3"
+    }
+  ]
+}
+```
+
+Don't forget to **Save**.
+
+All machines configured with your fragment, will update when they next check for configuration updates.
```
Comment (sguequierre) [inline docs/how-tos/deploy-packages.md:231]:
```suggestion
All machines configured with your fragment will update when they next check for configuration updates.
```

File: docs/how-tos/deploy-packages.md [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+
+## Next steps
+
+If you are setting up a larger fleet, you can use fragments to configure many machines and you can use Viam's provisioning manager, Viam Agent to automate the process of setting up machines with your fragments:
```
Comment (sguequierre) [inline docs/how-tos/deploy-packages.md:235]:
```suggestion
If you are setting up a larger fleet, you can use fragments to configure many machines and you can use Viam's provisioning manager, Viam Agent, to automate the process of setting up machines with your fragments:
```

File: docs/how-tos/deploy-packages.md [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+Now that the configuration is saved as a fragment, you can delete the machine you created in step 1.
+We only created this machine to easily generate the JSON config for the fragment.
+
+{{% /tablestep %}}
+{{< /table >}}
+
+## Add the fragment to multiple machines
+
+With your fragment created, you can add it to as many machines as you'd like:
+
```
Comment (npentrel) [inline docs/how-tos/deploy-packages.md:84]:
```suggestion

{{< alert title="Tip" color="tip" >}}
You can add multiple fragments to one machine.
{{< /alert >}}

```

**Review by shawnbmccarthy — COMMENTED** [review]
LGTM: might want to add that we support deploying multiple fragments

**Pull request merged**